mlflow/server/__init__.py

```diff
@@ -44,6 +45,12 @@ def serve_artifacts():
     return get_artifact_handler()
 
 
+# Serve the "get-model-version-artifact" route.
+@app.route(_add_static_prefix('/get-model-version-artifact'))
```

Comment:
To maintain some consistency with the naming pattern for other model version APIs, can we call this `/model-versions/get-artifact`?